python/pyspark/sql/readwriter.py

```diff
@@ -329,7 +329,8 @@ def csv(self, path, schema=None, sep=None, encoding=None, quote=None, escape=Non
                                          being read should be skipped. If None is set, it uses
                                          the default value, ``false``.
         :param nullValue: sets the string representation of a null value. If None is set, it uses
-                          the default value, empty string.
+                          the default value, empty string. Since 2.0.1, this ``nullValue`` param
```

Comment:
I think you can omit the "since x.y.z" in this PR. The new text will be in the docs for the version it applies to and not earlier ones.